package.json

```diff
@@ -203,4 +203,4 @@
       "eslint --fix --max-warnings=0"
     ]
   }
-}
+}
```

Comment:
Could you make sure there’s a new line here so this doesn’t get changed unnecessarily?